.github/workflows/docker_tag_check.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker Tag Version Check
+on:
+  push:
+    tags:
+      - '*'
+  repository_dispatch:
+    types: [one-app-docker-tag-check]
+jobs:
+  one-app-docker-tag-check:
+    name: Docker Tag Version Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
Do we need `actions/setup-node@v1`? I think we can remove this step..

Reply:
Maybe it's only needed in `.github/workflows/publish_statics_npm.yml` workflow? Could be missing something

Reply:
This a good point, we no longer need this. Thank you

Reply:
removed and updated this 3a3151f560650b7767ec1cd6f025ae331ad041aa e09fa8a5ddf77bce4f90ebfa42b9691040f8b356 a0c5c74f290d63b7be6c8b53245bc372996f404a